lib/inc/cpp-pcp-client/valijson/rapidjson_adapter.hpp

```diff
@@ -0,0 +1,785 @@
+/**
```

Comment:
This file is copied from the zipped valijson, and changed rapidjson::Value to ValueType. Comparing it against the original is probably the easiest way to review it.